pyproject.toml

```diff
@@ -13,9 +13,10 @@ description = "Logging/encoding/decoding using CLP's IR stream format"
 readme = "README.md"
 requires-python = ">=3.6"
 dependencies = [
+    "backports.zoneinfo >= 0.2.1; python_version<'3.9'",
     "clp-ffi-py >= 0.0.9",
-    "python-dateutil >= 2.7.0",
     "typing-extensions >= 3.7.4",
+	"tzlocal >= 5.2",
```

Comment:
btw, according to the document, this lib needs Python >= 3.8. We need to support Python3.7.
PyPI: https://pypi.org/project/tzlocal/

Reply:
Let's drop it to 5.1 then. That seems to be the last version that supports Python 3.7.

Reply:
> https://pypi.org/project/tzlocal/5.1/